manifest/product-config.json

```
@@ -708,22 +708,42 @@
             "start_build": 7,
             "trigger_blackduck": true
         },
-        "manifest/3.3.xml": {
+        "manifest/3.3.0.xml": {
+            "do-build": false,
             "go_version": "1.24.4",
-            "interval": 120,
+            "interval": 1440,
             "production": true,
             "release": "3.3.0",
             "release_name": "Couchbase Sync Gateway 3.3.0",
-            "start_build": 285,
+            "start_build": 287,
             "trigger_blackduck": true
         },
-        "manifest/4.0.xml": {
+        "manifest/3.3.xml": {
             "go_version": "1.24.4",
             "interval": 120,
             "production": true,
+            "release": "3.3.0.1",
+            "release_name": "Couchbase Sync Gateway 3.3.0.1",
+            "start_build": 1,
+            "trigger_blackduck": true
+        },
+        "manifest/4.0.0.xml": {
+            "do-build": false,
+            "go_version": "1.24.4",
+            "interval": 1440,
+            "production": true,
             "release": "4.0.0",
             "release_name": "Couchbase Sync Gateway 4.0.0",
-            "start_build": 243,
+            "start_build": 246,
```

```suggestion
            "start_build": 247,
```
I think this should be 247 based on latestbuilds?